scim2-sdk-common/src/main/java/com/unboundid/scim2/common/messages/PatchOperation.java

```diff
@@ -440,7 +439,7 @@ private void applyAddWithValueFilter(
       ObjectNode matchedValue = null;
       if (!APPEND_NEW_PATCH_VALUES_PROPERTY)
       {
-        matchedValue = fetchExistingValue(attribute, valueFilter, attributeName);
+        matchedValue = fetchExistingValue(attr, valueFilter, attributeName);
```

Comment:
I mostly renamed this variable because I didn't notice this line exceeds 80 characters when I first wrote it.